packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -38,6 +38,8 @@ class IconButton extends StatelessWidget {
   const IconButton({
     Key key,
     this.size: 24.0,
+    this.padding: const EdgeInsets.all(8.0),
+    this.alignment: FractionalOffset.centerLeft,
```

Comment:
Does that mean that the icon is not centered in the tap target? I think will feel really weird...